cookbook/integrations/gcp/bigquery/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.8-slim-buster
+LABEL org.opencontainers.image.source https://github.com/flyteorg/flytesnacks
+
+WORKDIR /root
+ENV VENV /opt/venv
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONPATH /root
+
+# This is necessary for opencv to work
+RUN apt-get update && apt-get install -y libsm6 libxext6 libxrender-dev ffmpeg build-essential curl
```

Comment:
nit: should we remove opencv dependencies?

Reply:
Done removed the dependencies but kept curl as its needed for installing gcloud sdk